common-data/loc-mailing-choices.json

```diff
@@ -0,0 +1,4 @@
+[
+    ["WE_WILL_MAIL", "Yes, have JustFix.nyc mail this letter on my behalf."],
+    ["USER_WILL_MAIL", "No thanks, I'll mail it myself."]
+]
```

Comment:
I could have just made this field a boolean, but it was easier to make it a choice field with two choices... I guess it also makes room for more options, just in case we ever want to provide more options.